nbextensions/bulkImportCell/testAppObj-ci.js

```diff
@@ -0,0 +1,472 @@
+define(['./jobsData'], (JobsData) => {
```

Comment:
Convert the old JSON file to JS, and populate the child jobs using the data from `jobsData.js`